styles.css

```diff
@@ -397,66 +383,163 @@ h1 {
   max-width: 400px;
 }
 
-@media screen and (max-width: 600px) {
-  .hamberger {
+@media only screen and (min-width: 600px) {
+  .header {
+    align-items: center;
+    background-color: #fff;
     display: flex;
-    position: absolute;
-    right: 25px;
-    top: 25px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  nav {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
   }
 
   .menu {
     align-items: center;
-    background-color: blueviolet();
-    display: none;
-    flex-direction: column;
-    height: 100%;
-    justify-content: start;
-    padding-top: 65px;
-    position: fixed;
-    right: 0;
-    top: 0;
-    width: 300px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    transition: 0.5s;
   }
 
   .menu li {
-    width: 100%;
+    list-style-type: none;
   }
 
   .menu li a {
-    box-shadow: 0 1px 0 rgba(255, 255, 255, 0.1) inset;
-    font-size: 24px;
-    padding: 30px;
+    color: #344563;
+    display: block;
+    font-size: 15px;
+    line-height: 1;
+    padding: 40px 25px;
+    transition: 0.3s;
+  }
+
+  .menu li a:hover {
+    box-shadow:
+      0 -5px 0 #fff inset,
+      500px 0 0 rgba(201, 42, 250, 0.2) inset;
+    padding: 35px 25px 45px;
+  }
+
+  .hamberger {
+    cursor: pointer;
+    display: none;
+    flex-direction: column;
+    height: 20px;
+    justify-content: space-around;
+    transition: 0.3s;
+    width: 40px;
+  }
+
+  .line {
+    background-color: #6070ff;
+    border: 2px solid #6070ff;
+    border-radius: 2px;
+    margin: 3px;
   }
 
   .logo {
-    margin-top: 10px;
+    height: 40px;
+    justify-content: left;
+    padding: 0 30px;
   }
 
   .portfolio {
-    background-image: url('./image/header-shapes\ mobile.png');
+    background-image: url('./image/Header_bg.svg');
+    padding: 20%;
   }
 
-  .info {
-    width: 80vw;
+  h1 {
+    font-size: 48px;
+  }
+
+  .description {
+    font-size: 20px;
+  }
+
+  /* Work section */
+
+  .work {
+    margin-top: 140px;
+    padding: 0;
+  }
+
+  .project {
+    margin: 30px auto;
+    width: 100%;
+  }
+
+  .project:nth-child(even) {
+    flex-direction: row-reverse;
   }
 
   .snap-shoot {
+    max-width: 550px;
     width: 80vw;
   }
 
   .snap-shoot img {
     width: 100%;
   }
 
-  .lang li {
```

Comment:
- [ ] Kindly align your tech languages properly as we have it in the figma design for the desktop version, this will show that we can work perfectly with any given template. 😄 

##Figma Design
![astack](https://user-images.githubusercontent.com/103386183/206164985-cdec7dc7-6fe0-4fee-8828-001c51aec659.png)